src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.spec.ts

```diff
@@ -48,6 +49,7 @@ describe('DotEditLayoutGridComponent', () => {
                 DotConfirmationService,
                 DotEditLayoutService,
                 PaginatorService,
+                DotEventsService,
```

Comment:
do you need add a new test where this service is used?, you have to create a mock of EventService and tigger the listen method for 'dot-side-nav-toggle' event and then check that the this.ngGrid.triggerResize method is called